config/v1/types_infrastructure.go

```diff
@@ -562,6 +573,21 @@ type BareMetalPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// loadBalancerType defines if an user-managed or openshift-managed loadbalancer
+	// is to be used for the openshift API and Ingress endpoints.
+	// When set to OpenShiftManaged the static pods in charge of API and Ingress traffic load-balancing
+	// defined in the machine config operator will be deployed.
+	// When set to UserManaged these static pods will not be deployed and it is expected that
+	// the load balancer is configured out of band by the deployer.
+	// When omitted, this means no opinion and the platform is left to choose a reasonable default
```

Comment:
Because you're using a kubebuilder default, you can never change that default later. So in this case, where you say the platform is left to choose a reasonable default, that's true, but the "The current default value is OpenShiftManaged" is possibly misleading as it hints that the value could change later down the line, which it can't.

So I think we need to say
```suggestion
	// When omitted, this means no opinion and the platform is left to choose a reasonable default.
	// The default value is OpenShiftManaged.
```